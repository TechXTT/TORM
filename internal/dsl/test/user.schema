entity.User().
    Field("id", "uuid.UUID").NotNull().PrimaryKey().
    Field("Username", "string").
    Field("Email", "string").
    Field("Password", "string").
    Field("CreatedAt", "time.Time").Default("now()").
    Field("UpdatedAt", "time.Time").Default("now()").
    Field("DeletedAt", "time.Time").Default(nil).