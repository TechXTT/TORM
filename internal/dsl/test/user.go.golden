package test

import (
	uuid "github.com/google/uuid"
	"time"
)

type User struct {
	id        uuid.UUID `db:"id"`
	Username  string    `db:"username"`
	Email     string    `db:"email"`
	Password  string    `db:"password"`
	CreatedAt time.Time `db:"createdat"`
	UpdatedAt time.Time `db:"updatedat"`
	DeletedAt time.Time `db:"deletedat"`
}
